### ISSUE TITLE
ändern/löschen und noch besser 'erledigen' von Kommentaren

### ISSUE THREAD

**Issue by melfja** [maplesteve/swaggertalk#27]
die Papierkorb und editieren Symbole sind offensichtlich noch ohne Funktion. das wäre nett.
Noch wichtiger fände ich ein 'erledigt' Knopf:  dann kann man den Kommentar noch sehen, weiss aber das er nicht mehr relevant ist.
Ideal wäre es die erledigten 'auszublenden'.

**Comment by maplesteve** [comment]
O.k. (die beiden Symbole sind in der Tat noch ohne Funktion)
Siehst Du hier die Notwendigkeit, Threads in den Kommentaren zu unterstützen - wenn z.B. mehrere 'Kritikpunkte' unter einem Punkt aufgeschrieben und diskutiert werden, dann wird das in der rein linearen Darstellung wahrscheinlich unübersichtlich.

**Comment by melfja** [comment]
Ja, threads wären nice. Da kommentare schon jetzt sehr lokal sind, wäre das nicht so drängend. 
In erster stufe wäre eine erledigt  Funktion ausreichend um den Überblick zu behalten
